lambdaclass/mirra_backend
# Limit amount of bounties sent to the client

## Motivation

We're sending every bounty in the backend db when the client is expecting to receive only 3 
Also the client need the selecting stage timer and the start game timer


## Summary of changes

Limit amount of bounties sent to the client
Send needed timers

## How to test it?

Use this client pr -> https://github.com/lambdaclass/champions_of_mirra/pull/1791
1. run `GameBackend.CurseOfMirra.Config.import_quest_descriptions_config` in the backend terminal
2. wait for the bounty fetcher to retrieve bounties, 30 sec max
3. Start a match and you should see the 3 bounties that are created in the quest_descriptions.json file

## Checklist
